kolibri/plugins/learn/assets/src/views/LibraryPage/index.vue

```diff
@@ -402,6 +405,7 @@
       };
     },
     computed: {
+      ...mapGetters(['isLearnerOnlyImport']),
```

Comment:
This is fine - in future, we'll try to move more towards using the core `useUser` composable that wraps this, so we can deprecate and remove the core vuex state.